github-dark.user.css

```diff
@@ -172,6 +172,7 @@
   } EOT;
 }
 @advanced text font-choice "Code Font" "Menlo"
+@advanced text font-size-choice "Code Font Size" "12px"
```

Comment:
```suggestion
@advanced text font-size-choice "Code Font Size" "github default"
```

Reply:
Did you try setting the font-size to `initial` or `inherit`?